### PR TITLE
mlst 2.23.0 with updated databases (January 2024)

### PR DESCRIPTION
**Pull request by cimendes**
This PR updates mlst version 2.23.0 with the latest database available from pubmlst as of January 2024. No other modifications were made to the dockerfile. The README has been updated with the date of the update. 

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in the same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: mlst/2.23.0-2024-01/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+LABEL dockerfile.version="1"
+LABEL software="mlst"
+LABEL software.version="${MLST_VER}"
+LABEL description="Scan contig files against PubMLST typing schemes"
+LABEL website="https://github.com/tseemann/mlst"
+LABEL license="https://github.com/tseemann/mlst/blob/master/LICENSE"
+LABEL maintainer="Inês Mendes"
+LABEL maintainer.email="ines.mendes@theiagen.com"
+
+# copy README for posterity
```
Comment (kapsakcj) [inline mlst/2.23.0-2024-01/Dockerfile:16]: Oooh, I like this! Never thought of baking the README.md into the image! 

`COPY` Adds an additional layer, but I don't think this is a big deal. Might be slightly better to move this bit of code to the end of the `app` layer, right before the `FROM app as test` line 71. This way the caching of the `apt-get` layers may be used between different docker images.

**Comment by kapsakcj** [comment]
No, I think are good to merge and deploy. Apologies for not giving the final approval. thanks for the reminder

**Review by kapsakcj — APPROVED** [review]

**Comment by erinyoung** [comment]
Thanks!

I'm going to 
- merge this PR
- deploy to dockerhub and quay using the tag 2.23.0-2024-01
- overwrite the 'latest' tag

**Comment by erinyoung** [comment]
Thank you for your contribution!

You can check the status of the deploy here: https://github.com/StaPH-B/docker-builds/actions/runs/7478119120

**Pull request merged**